cmd/root/user_agent_command_exec_id.go

```diff
@@ -7,8 +7,10 @@ import (
 	"github.com/google/uuid"
 )
 
+var cmdExecId = uuid.New().String()
+
```

Comment:
This is a global now. It should be initialized once per command and can be set on the context.

Reply:
Done in https://github.com/databricks/cli/pull/2439. Will update this PR once the other one is merged.

Reply:
Fixed.